main/SS/Formula/PTG/OperandPtg.cs

```diff
@@ -37,7 +37,8 @@ public OperandPtg Copy()
         {
             try
             {
-                return (OperandPtg)Clone();
+                // REMOVE-REFLECTION: After careful inspection, MemberwiseClone() should be enough for all built-in OpreandPtgs.
+                return (OperandPtg)MemberwiseClone();
             }
```

Comment:
All derived classes of `OperandPtg` holds only value types (some unused members are not, but that's fine).